nixos/modules/services/databases/arangodb.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.arangodb;
+
+in {
+
+  ###### interface
+
+  options = {
+    services.arangodb = {
+      enable = mkEnableOption "ArangoDB server";
+
+      dataDir = mkOption {
+        type = types.path;
+        example = "/var/db/arangodb";
+        default = "/var/db/arangodb";
```

Comment:
Having both `example` and `default` on the same value seems redundant. I think `example` should just be omitted in this case.